batch/batch/front_end/templates/billing_limits.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout.html" %}
+{% block title %}Billing Limits{% endblock %}
+{% block content %}
+<h1>Billing Project Limits</h1>
+<div class='flex-col' style="overflow: auto;">
+    <table class="data-table" id="billing_limits">
+        <thead>
+        <tr>
+            <th>Billing Project</th>
+            <th>Accrued Cost</th>
+            <th>Limit</th>
+            <th></th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for row in billing_limits %}
+        <form action="{{ base_path }}/billing_limits/{{ row['billing_project'] }}/edit" method="POST">
```

Comment:
I'm pretty sure this is not valid HTML. Forms shouldn't be children of tables. The fix seems easy here: move the form and the hidden input all inside the `td` that contains the limit

Reply:
Yeah, I tried this:
```
<!DOCTYPE html>
<html lang="en">
<head><title>test</title></head>
<body>
<table>
<tbody>
<form>
<input type="hidden" name="_csrf" value="123">
<tr>
  <td>
    <input type="text" required name="limit" value="1.0">
  </td>
</tr>
<tr>
  <td>
    <input type="text" required name="limit" value="2.0">
  </td>
</tr>
</input>
</form>
</tbody>
</table>
</body>
</html>
```

in the HTML validator: https://validator.w3.org/nu/#textarea

Reply:
This validated:

```
<!DOCTYPE html>
<html lang="en">
<head><title>test</title></head>
<body>
<table>
<tbody>
<tr>
  <td>
    <form>
    <input type="hidden" name="_csrf" value="123">
    <input type="text" required name="limit" value="1.0">
    </form>
  </td>
</tr>
<tr>
  <td>
    <form>
    <input type="hidden" name="_csrf" value="123">
    <input type="text" required name="limit" value="2.0">
    </form>
  </td>
</tr>
</tbody>
</table>
</body>
</html>
```